src/AblyFactory.php

```diff
@@ -34,8 +34,8 @@ public function make($clientOptions = null)
      */
     protected function createInstance($clientOptions)
     {
-        AblyRest::setLibraryFlavourString('laravel');
-
+        $laravelVersion = app()->version();
+        AblyRest::setLibraryFlavourString('laravel/'.$laravelVersion);
```

Comment:
I think here we should be using the new API (i.e. calling `setAblyAgentHeader`) and updating `composer.json` to require a version of `ably-php` which includes that new API (and if we haven't released a version with that change in it yet, then we should do that first).

Reply:
Yeah, I am planning to do it, once we have a new release ready 👍

Reply:
Done